deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -42,6 +42,9 @@ class TrainTestFeatureDrift(TrainTestCheck, ReduceMixin):
     We also support Population Stability Index (PSI).
     See https://www.lexjansen.com/wuss/2017/47_Final_Paper_PDF.pdf.
 
+    For categorical variables, it is recommended to use Cramer’s V, unless your variable includes categories with a
+    small number of samples, where PSI would perform better. However, in cases of a variable with a large number of
+    unique categories, it is still recommended to use Cramer’s V.
```

Comment:
Not clear enough to me what "unique categories" are. I guess you meant "with only a single occurrence"?

Also, "include categories with a small number of samples". Any amount of such categories?

Reply:
1. Actually not what I meant. How's now?
2. I don't feel comfortable enough to say. It is common to say that Cramer's V is less accurate when <5 samples, but it's a continuous value of course and fluctuates at higher values as well. Changed to fit this. WDYT?